definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -5289,7 +5289,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorSubmitAOS",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "R"
+    "CRUD": "CRUD"
   },
```

Comment:
There are many AOS events currently present with caseworker_beta set to "R". 
Is this event all that is required for this story?
@yarpo

Reply:
I have no idea. We need to use the event that will be triggered for AOS pack 2. I have never worked or refined stories about it :( [or at least I don't remember event's name)

Reply:
I think your story wants you to do them all, @jacmurphy.
@yarpo, not sure we want to release this until the very end of the epic.

Reply:
Thanks @jvictorcf . Ill update them now